src/pages/Projects/components/RoleCard/RoleCard.tsx

```diff
@@ -181,6 +184,39 @@ export default function RoleCard({
                 Timeline
               </Text>
             </Td>
+            <Td colSpan={6}>
+              <Box key={role.id}>
+                <Flex
+                  key={role.skills[0].id}
+                  alignItems="center"
+                  borderBottom="1px solid rgba(0,0,0,0,04)"
+                  minHeight="50px"
+                >
+                  {timeline.map((item: TimelineRange, index: number) => {
+                    return (
+                      <Box
+                        textAlign="center"
+                        alignSelf="stretch"
+                        key={`gantt-cell-project-${role.id}-${index}`}
+                        backgroundColor={
+                          index % 2 === 0 ? "rgba(0,0,0,.04)" : "transparent"
+                        }
+                        flex="1"
+                      >
+                        <Flex marginTop="14px" flexDirection="column">
+                          <GanttCell
+                            roleAssignments={getRolesAsRow(role)}
+                            timeline={timeline}
+                            index={index}
+                            skill={role.skills[0]}
```

Comment:
this locks the skill to the first element in the array, is that ok?

Reply:
yup. should only ever be one skill